src/app/[owner]/[repo]/slides/page.tsx

```
@@ -1249,8 +1247,7 @@ Please return ONLY the HTML with no markdown formatting or code blocks. Just the
               <div
                 className={`${isFullscreen ? 'w-full h-full' : 'w-full max-w-[1280px] aspect-[16/9]'} flex items-center justify-center overflow-hidden`}
               >
-                {/* Include Font Awesome for icons */}
-                <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.4.0/css/all.min.css" />
+                {/* Font Awesome CDN removed - using react-icons instead */}
                 <div className="w-full h-full" dangerouslySetInnerHTML={{ __html: slides[currentSlideIndex]?.html || '' }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)